### PR TITLE
Submitting a blog about IstioCon program

### PR DESCRIPTION
**Pull request by macruzbar**
[ ] Configuration Infrastructure
[X ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/blog/2021/istiocon-2021-program/.spelling.md [hunk 1]
```diff
@@ -0,0 +1,766 @@
+# markdown-spellcheck spelling configuration file
```
Comment (ericvn) [inline content/en/blog/2021/istiocon-2021-program/.spelling.md:1]: We typically use the .spelling in the root directory of the repo.

**Comment by nrjpoddar** [comment]
I would recommend adding a blurb about Istio 2021 roadmap if you can.

**Review by craigbox — CHANGES_REQUESTED** [review]
`content/en/blog/2020/istioconlogo.jpg` => that should be 2021?  It doesn't appear to be used anyway, so you may wish to remove it.

(I can't see the preview sorry, I've pinged Docs WG on Slack to see why that is)

File: content/en/blog/2021/istiocon-2021-program/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+---
+title: "IstioCon 2021: Schedule Is Live!"
+description: Learn about sessions, panels, workshops and more on the IstioCon website.
+publishdate: 2020-02-12
```
Comment (craigbox) [inline content/en/blog/2021/istiocon-2021-program/index.md:4]:
```suggestion
publishdate: 2021-02-16
```

**Comment by frankbu** [comment]
This PR seems time sensitive, so I quickly fixed it up in #8983.

**Pull request closed without merging**